File: OPP/JS/cars/UberX.js
import Car from "../cars/Car.js";

export default class UberX extends Car {
  constructor(license, driver, brand, model) {
    super(license, driver);
    this.brand = brand;
    this.model = model;
  }

  printDataUberX() {
    console.log(`Driver: ${this.driver}`);
    console.log(`License: ${this.license}`);
    console.log(`Car brand: ${this.brand}`);
    console.log(`Car model: ${this.model}`);
  }
}

File: OPP/Java/UberX.java
class UberX extends Car {
  String brand;
  String model;

  public UberX(String license, Account driver, String brand, String model) {
    super(license, driver);
    this.brand = brand;
    this.model = model;
  }

  void getDataUberX() {
    System.out.println("Car license: " + license + " Driver: " + driver.name + " Brand: " + brand + " Model: " + model);
  }
}
